srcpkgs/wvkbd/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'wvkbd'
+pkgname=wvkbd
+version=0.7
+revision=1
+build_style=gnu-makefile
+make_build_args="CC=$CC"
```

Comment:
No need for this with `make_use_env=1`, the Makefile looks fine enough to honor most variables `xbps-src` sets, [for `LDFLAGS` there is a PR](https://github.com/jjsullivan5196/wvkbd/pull/27) as well and the patch can be carried here till it's merged.

Reply:
should this be `make_use_env=yes?`